### PR TITLE
Awesomebooks with ES6

### PR DESCRIPTION
**Pull request by monika-soam**
**Setup project:-**

- Created a new directory and initialized it with git.
- Used npm init -y command to create package.json file.
- Created the entry point for your JavaScript code called index.js
- Created index.css file for styles.
- Created the main index.html file. Linked index.js and index.css files there.
- Created a directory called modules.
- Used ES6 syntax.
- Divided code into modules. Save each module in a separate file in the modules dir. Imported modules in the index.js file.
- Refactored methods and functions to arrow functions.

**Displayed the current date below the navigation bar:-**
- For this requirement used a library Luxon
- Downloaded the script and saved it to the project and Imported it as a module
- No linters error.
- Followed the correct GitHub flow and also documented the work.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+const displayBook = document.getElementById('bookslist');
+const container = document.getElementById('container');
+const addbook = document.getElementById('addbook');
+const contact = document.getElementById('contact');
+const addnewbook = document.getElementById('addnewsec');
+const listbook = document.getElementById('listsec');
+const showcontact = document.getElementById('contactsec');
+const removeButtons = document.getElementsByClassName('btn-remove');
+
+function removeMyBook(i) {
```
Comment (Amrhub) [inline index.js]: - [x] As this is a practice for es6 model syntax let's kindly convert this to an arrow function.

File: index.js [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+    </tr>`;
+  }
+  for (let i = 0; i < removeButtons.length; i += 1) {
+    removeButtons[i].addEventListener('click', () => {
+      removeMyBook(i);
+    });
+  }
+}
+
+function showMyBooks() {
```
Comment (Amrhub) [inline index.js]: - [x] This one as well ^_^

File: modules/addBook.js [hunk 3]
```diff
@@ -0,0 +1,11 @@
+export default function addBook(title, author) {
```
Comment (Amrhub) [inline modules/addBook.js]: - [x] This one as well ^_^

File: modules/dateTime.js [hunk 4]
```diff
@@ -0,0 +1,27 @@
+function getNumberSuffix(num) {
```
Comment (Amrhub) [inline modules/dateTime.js]: - [x] This one as well ^_^

File: modules/dateTime.js [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+    case '2':
+      return '<sup>nd</sup>';
+    case '3':
+      return '<sup>rd</sup>';
+    default:
+      return '<sup>th</sup>';
+  }
+}
+
+export default function dateTime() {
```
Comment (Amrhub) [inline modules/dateTime.js]: - [x] This one as well ^_^

File: modules/listBooks.js [hunk 6]
```diff
@@ -0,0 +1,7 @@
+export default function bookList() {
```
Comment (Amrhub) [inline modules/listBooks.js]: - [x] This one as well ^_^

File: modules/removeBook.js [hunk 7]
```diff
@@ -0,0 +1,11 @@
+export default function removeBook(i) {
```
Comment (Amrhub) [inline modules/removeBook.js]: - [x] This one as well ^_^

**Review by Amrhub — CHANGES_REQUESTED** [review]
Hi @monika-soam,  👋🏻 

Good job so far! 👍🏻 
There are some issues that you still need to work on to go to the next project but you are almost there!
These changes aim to add some awesomeness to your awesome project and code skills.

## To Highlight ✔️ 
Highly organized and clean code ✔️ 👨🏻‍💻 
Professional documentation ✔️ 
Correct workflow ✔️ 
Meaningful commit messages ✔️ 


## Required Changes ♻️
_Check the comments under the review._


- [ ] I really like the project UI design, I highly recommend we use ES6 arrow functions instead of regular functions. I will point out all that I could find to help you not miss any of them

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.js (hunk 1 above)
Comment (monika-soam) [reply index.js]: Fixed it.

File: index.js (hunk 2 above)
Comment (monika-soam) [reply index.js]: Fixed it.

File: modules/addBook.js (hunk 3 above)
Comment (monika-soam) [reply modules/addBook.js]: Fixed it.

File: modules/dateTime.js (hunk 4 above)
Comment (monika-soam) [reply modules/dateTime.js]: Fixed it.

File: modules/dateTime.js (hunk 5 above)
Comment (monika-soam) [reply modules/dateTime.js]: Fixed it.

File: modules/listBooks.js (hunk 6 above)
Comment (monika-soam) [reply modules/listBooks.js]: Fixed it.

File: modules/removeBook.js (hunk 7 above)
Comment (monika-soam) [reply modules/removeBook.js]: Fixed it.

File: index.js (hunk 1 above)
Comment (monika-soam) [reply index.js]: Thank you for your help as pointed out the specific lines

File: modules/listBooks.js (hunk 6 above)
Comment (monika-soam) [reply modules/listBooks.js]: Thank you for your help

**Review by gemmen29 — APPROVED** [review]
Hi @monika-soam, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Optional suggestions

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**